core/src/main/scala/kafka/server/AddPartitionsToTxnManager.scala

```diff
@@ -39,13 +39,33 @@ object AddPartitionsToTxnManager {
 
   val VerificationFailureRateMetricName = "VerificationFailureRate"
   val VerificationTimeMsMetricName = "VerificationTimeMs"
+
+  def produceRequestVersionToTransactionSupportedOperation(version: Short): TransactionSupportedOperation = {
+    if (version > 11) {
+      addPartition
+    } else if (version > 10) {
+      genericError
```

Comment:
This  looks like we're returning an error.  Can we change the names, maybe `genericErrorSupported` and etc.?

Reply:
Done.